### PR TITLE
Add snowflake proxy ansible role

### PR DESCRIPTION
**Pull request by anadahz**

### CODE REVIEW

File: ansible/roles/snowflake/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  until: sf_nat_type.rc == 0
+
+- name: Show snowflake proxy NAT type
+  debug:
+    msg: "{{ sf_nat_type.stdout }}"
+
+- name: Assert snowflake proxy NAT is unrestricted
+  assert:
+    that: "'unrestricted' in sf_nat_type.stdout"
+    fail_msg: "[ERR] snowflake proxy NAT type is not unrestricted"
```
Comment (francisco-core) [inline ansible/roles/snowflake/tasks/main.yml:52]: So basically we get a test server and make sure that this passes, correct?

**Comment by francisco-core** [comment]
```
 TASK [snowflake : Set the iptables policy for the INPUT chain to ACCEPT] *******
  failed: [debian11-instance] (item=v4) => {"ansible_loop_var": "item", "changed": false, "item": "v4", "msg": "Failed to find required executable \"iptables\" in paths: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"}
  failed: [debian11-instance] (item=v6) => {"ansible_loop_var": "item", "changed": false, "item": "v6", "msg": "Failed to find required executable \"ip6tables\" in paths: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"}
```
I guess we have to install `iptables`

File: ansible/roles/snowflake/tasks/main.yml (hunk 1 above)
Comment (anadahz) [reply ansible/roles/snowflake/tasks/main.yml:52]: > So basically we get a test server and make sure that this passes, correct?

Correct.

**Comment by anadahz** [comment]
> ```
>  TASK [snowflake : Set the iptables policy for the INPUT chain to ACCEPT] *******
>   failed: [debian11-instance] (item=v4) => {"ansible_loop_var": "item", "changed": false, "item": "v4", "msg": "Failed to find required executable \"iptables\" in paths: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"}
>   failed: [debian11-instance] (item=v6) => {"ansible_loop_var": "item", "changed": false, "item": "v6", "msg": "Failed to find required executable \"ip6tables\" in paths: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"}
> ```
> 
> I guess we have to install `iptables`

Fixed in: https://github.com/PrivacyLx/devops/pull/98/commits/ab190d576d29405e1b83c56e40c88be52098a2de

**Comment by francisco-core** [comment]
Awesome work! Thanks a lot!

**Pull request merged**